README.md

```diff
@@ -16,19 +17,19 @@
 
 ## Introduction
 
-Regula is a very light (less than 100 lines of bash) framework around [opa] and
-[terraform] that allows you to more easily implement pre-flight policy checks,
-by providing a rigid structure for rules.
+Regula is a tool that evaluates Terraform infrastructure-as-code for potential security misconfigurations and compliance violations prior to deployment.
```

Comment:
@fugue-chris Should we mention Fugue anywhere here?

Reply:
No, I intentionally left Fugue out of it